TaskStatusReporter/TaskStatusReporter.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+# encoding: utf-8
+'''
+EC2Reporter -- Simple EC2 status reporting tool for EC2SpotManager
+
+Provide process and class level interfaces to send simple textual
+status reports to EC2SpotManager.
+
+@author:     Jesse Schwartzentruber (:truber)
+
+@license:
+
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@contact:    jschwartzentruber@mozilla.com
+'''
+
+# Ensure print() compatibility with Python 3
+from __future__ import print_function
+
+import argparse
+import functools
+import os
+import random
+import sys
+import time
+
+from fasteners import InterProcessLock
+import requests
+
+from FTB.ConfigurationFiles import ConfigurationFiles  # noqa
+from Reporter.Reporter import Reporter, remote_checks  # noqa
+
+__all__ = []
+__version__ = 0.1
+__date__ = '2014-10-01'
+__updated__ = '2014-10-01'
+
+
+class TaskStatusReporter(Reporter):
+    @functools.wraps(Reporter.__init__)
+    def __init__(self, *args, **kwargs):
+        kwargs.setdefault('tool', 'N/A')  # tool is required by remote_checks, but unused by TaskStatusReporter
```

Comment:
Is there a particular reason for the decision to remove the tool association that we had for EC2?

Reply:
This is copy-paste from EC2Reporter. `Tool` is already not used by EC2SpotManager, only CovManager and CrashManager.